apps/build-onchain-apps/src/components/footer/Footer.tsx

```diff
@@ -0,0 +1,75 @@
+import { GitHubLogoIcon } from '@radix-ui/react-icons';
+import NextLink from 'next/link';
+import { NavbarLink } from '../header/Navbar';
+
+export default function Footer() {
+  return (
+    <footer className="h-[22.1875rem] bg-[#141519] py-12">
```

Comment:
Two things:
1. let's try to have rem that has not extra digits, like the simple the better
2. bg, we should create in the `tailwind.config.ts` file shared bg color, and add this one there. So it's more clean to use it.

Reply:
@Zizzamia done at d0d419b